xbmc/filesystem/FavouritesDirectory.cpp

```diff
@@ -197,8 +197,15 @@ std::string CFavouritesDirectory::GetExecutePath(const CFileItem &item, const st
       execute = StringUtils::Format("ActivateWindow(%s,%s,return)", contextWindow.c_str(), StringUtils::Paramify(item.GetPath()).c_str());
   }
   //! @todo STRING_CLEANUP
-  else if (item.IsScript() && item.GetPath().size() > 9) // plugin://<foo>
+  else if (item.IsScript() && item.GetPath().size() > 9) // script://<foo>
     execute = StringUtils::Format("RunScript(%s)", StringUtils::Paramify(item.GetPath().substr(9)).c_str());
+  else if (item.IsPlugin() && item.GetPath().size() > 9) // plugin://<foo>
+    execute = StringUtils::Format("PunPlugin(%s)", StringUtils::Paramify(item.GetPath().substr(9)).c_str());
```

Comment:
ah ok, now i got it, plugins get translated to ActivateWindow(xx,xx).... will fix.
(I like PunPlugin though, perhaps I will propose that in a follow-up PR)